packages/template/src/project.ts

```diff
@@ -3,5 +3,6 @@ import {makeProject} from '@motion-canvas/core';
 import example from './scenes/example?scene';
 
 export default makeProject({
+  experimentalFeatures: true,
```

Comment:
Is this intended to be set to true in the template project?

Reply:
I think so. This is an internal package meant for maintainers to test new features without having to set up npm links etc. so I think having them enabled is a better default